testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -278,6 +279,8 @@
 XYZ = resource_filename(__name__, 'data/2r9r-1b.xyz')
 XYZ_mini = resource_filename(__name__, 'data/mini.xyz')
 XYZ_five = resource_filename(__name__, 'data/five.xyz')
```

Comment:
You must also add an entry to `setup.py` so that files with the txyz and arc suffix are included in the tr ball. Otherwise they will be missing from the packages.